src/cosmosdb-preview/azext_cosmosdb_preview/_params.py

```diff
@@ -142,3 +142,53 @@ def load_arguments(self, _):
+        c.argument('cluster_name', options_list=['--cluster-name', '-c'], help="Cluster Name", required=True)
+
+    # Managed Cassandra Cluster
+    with self.argument_context('cassandra-managed-instance cluster node-status') as c:
+        c.argument('cluster_name', options_list=['--cluster-name', '-c'], help="Cluster Name", required=True)
+
+    # Managed Cassandra Cluster
+    with self.argument_context('cassandra-managed-instance cluster show') as c:
+        c.argument('cluster_name', options_list=['--cluster-name', '-c'], help="Cluster Name", required=True)
+
+    # Managed Cassandra Cluster
+    with self.argument_context('cassandra-managed-instance cluster delete') as c:
+        c.argument('cluster_name', options_list=['--cluster-name', '-c'], help="Cluster Name", required=True)
+
+    # Managed Cassandra Datacenter
+    with self.argument_context('cassandra-managed-instance datacenter create') as c:        
+        c.argument('cluster_name', options_list=['--cluster-name', '-c'], help="Cluster Name", required=True)
+        c.argument('data_center_name', options_list=['--data-center-name', '-d'], help="Datacenter Name", required=True)
+        c.argument('data_center_location', options_list=['--data-center-location', '-l'], help="Azure Location of the Datacenter", required=True)
+        c.argument('delegated_subnet_id', options_list=['--delegated-subnet-id', '-s'], help="The subnet we will inject the Network Interfaces of the data vms", required=True)
+        c.argument('node_count', options_list=['--node-count', '-n'], help="Node Count", required=True)
+
+    # Managed Cassandra Datacenter
+    with self.argument_context('cassandra-managed-instance datacenter update') as c:        
+        c.argument('cluster_name', options_list=['--cluster-name', '-c'], help="Cluster Name", required=True)
+        c.argument('data_center_name', options_list=['--data-center-name', '-d'], help="Datacenter Name", required=True)
+        c.argument('node_count', options_list=['--node-count', '-n'], help="Node Count")
+
+    # Managed Cassandra Datacenter
+    with self.argument_context('cassandra-managed-instance datacenter list') as c:
+        c.argument('cluster_name', options_list=['--cluster-name', '-c'], help="Cluster Name", required=True)
+
+    # Managed Cassandra Datacenter
+    with self.argument_context('cassandra-managed-instance datacenter show') as c:
+        c.argument('cluster_name', options_list=['--cluster-name', '-c'], help="Cluster Name", required=True)
+        c.argument('data_center_name', options_list=['--data-center-name', '-d'], help="Datacenter Name", required=True)
+
+    # Managed Cassandra Datacenter
+    with self.argument_context('cassandra-managed-instance datacenter delete') as c:
+        c.argument('cluster_name', options_list=['--cluster-name', '-c'], help="Cluster Name", required=True)
+        c.argument('data_center_name', options_list=['--data-center-name', '-d'], help="Datacenter Name", required=True)
```

Comment:
Same suggestion with `data_center_name`. You can define `data_center_name_type` and apply it to the whole `cassandra-managed-instance datacenter` command group.